pkg/apis/minio.min.io/v1/constants.go

```diff
@@ -72,11 +73,15 @@ const MinIOVolumeSubPath = ""
 // DefaultMinIOImage specifies the default MinIO Docker hub image
 const DefaultMinIOImage = "minio/minio:RELEASE.2020-07-22T00-26-33Z"
 
+// DefaultMinIOUpdateURL specifies the default MinIO URL where binaries are
+// pulled from during MinIO upgrades
+const DefaultMinIOUpdateURL = "https://dl.min.io/server/minio/release/" + runtime.GOOS + "-" + runtime.GOARCH + "/archive/"
+
 // DefaultMinIOAccessKey specifies default access key for Tenant
-const DefaultMinIOAccessKey = "AKIAIOSFODNN7EXAMPLE"
+const DefaultMinIOAccessKey = "ACCESS-KEY-EXAMPLE"
```

Comment:
Nit: We should not have a default password that scanners try to discovery lazy ops..

Reply:
yeah that's why renaming it as `-EXAMPLE` - not sure if we don't want a default perhaps fail if its not set?

Reply:
Yeah the operator can fail the instance creation if secret not set.

Reply:
Looks like its not used, removing it.